routes/web.php

```diff
@@ -52,9 +48,6 @@
 
     // old route redirects
     Route::permanentRedirect('poll-log', 'poller/log');
-    Route::get('settings/sub={tab}', function ($tab) {
```

Comment:
removing this route could break some external links.  It can be changed to use a controller.

Reply:
From where? It's a link to a settings page, no big deal. 
And like I wrote, it doesn't even work atm due to being processed after the real settings-route.

Reply:
When people deep link into LibreNMS (from their internal documentation for example), changing the structure will break those links.  This is probably most common with the device link however...

I see your point, I think it got re-organized.  I'm fine with removing this.